### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by ANDREYDEN**
- Adds Firebase to the app
- Log Out is broken, but I think we can fix it later

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Memories Keeper
+
+## Firebase
+
+### Auth
+
+Make sure to add the SHA-1 fingerprint to the Firebase console: 
+`keytool -list -v -keystore "%USERPROFILE%\.android\debug.keystore" -alias androiddebugkey -storepass android -keypass android`
```
Comment (alisondraV) [inline README.md]: This command didn't work for me. But this one did (with the password `android`):
`keytool -list -v \
-alias androiddebugkey -keystore ~/.android/debug.keystore`

**Review by alisondraV — COMMENTED** [review]
We might need to update README. And broken sign out seems worrying to me. But sure.

File: app/src/main/res/layout/activity_home.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        android:text="Home"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Button
+        android:id="@+id/btnLogout"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Log Out"
```
Comment (alisondraV) [inline app/src/main/res/layout/activity_home.xml:22]: Make sure to extract string resources and center TextViews with Buttons, please

Comment (ANDREYDEN) [reply app/src/main/res/layout/activity_home.xml:22]: I'm not doing this because this is not permanent

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -10,7 +10,10 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.MemoriesKeeper">
         <activity
-            android:name=".MainActivity"
+            android:name=".HomeActivity"
+            android:exported="true" />
+        <activity
+            android:name=".LoginActivity"
```
Comment (alisondraV) [inline app/src/main/AndroidManifest.xml]: Let's rename this to MainActivity

File: app/src/main/java/com/example/memorieskeeper/LoginActivity.java [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+    private void updateUI(boolean isSignedIn) {
+        setContentView(R.layout.activity_login);
+
+        findViewById(R.id.btnSignIn).setOnClickListener(view -> signIn());
+
+        if (isSignedIn) {
+            startActivity(new Intent(this, HomeActivity.class));
+        }
+    }
```
Comment (alisondraV) [inline app/src/main/java/com/example/memorieskeeper/LoginActivity.java]: Let's move `setOnClickListener` to `onCreate` method and inline `startActivity(new Intent(this, HomeActivity.class));` where it is needed.

**Review by alisondraV — CHANGES_REQUESTED** [review]

**Review by alisondraV — APPROVED** [review]

**Pull request merged**